apps/unlock/lib/controller.ex

```diff
@@ -23,6 +23,22 @@ defmodule Unlock.Controller do
     defstruct [:body, :headers, :status]
   end
 
+  defmodule Telemetry do
+    @moduledoc """
+    A quick place to centralize definition of tracing events and targets
+    """
+
+    def target_for_identifier(item_identifier) do
+      "proxy:#{item_identifier}"
+    end
+
+    def trace_request(item_identifier, request_type) do
+      :telemetry.execute([:proxy, :request, request_type], %{}, %{
```

Comment:
Donc ceci sera traité par `handle_event` dans `Transport.Telemetry` ?

Reply:
Tout à fait ; j'ai ajouté une noté dans 2c26e118